src/router.js

```diff
@@ -3,6 +3,9 @@ import routes from "./routes"
 import { NetworkCapability, NetworkCapabilityResult } from "./gql"
 import Vue from "vue"
 
+// Reset scroll
+Vue.prototype.$scrollToTop = () => window.scrollTo(0, 0)
```

Comment:
Why do you add it to the prototype? You can just add it as a method to `AppMenu`

Reply:
Totally right!